frontend/src/hooks/pims-api/useApiRequestWrapper.ts

```diff
@@ -107,7 +107,8 @@ export const useApiRequestWrapper = <
 
   useEffect(() => {
     if (invoke) {
-      wrappedApiRequest();
+      // If invoke is set then the function is parameterless.
+      wrappedApiRequest(undefined as Parameters<FunctionType>);
```

Comment:
this seems undesirable to me calling a function with undefined != calling with no parameters

Reply:
It only needs to happen here. In other cases the function parameters need to match the specified FunctionType, otherwise, it will not compile. In this case, the FunctionType is still generic so there is no way to know if it should work